Tests/test_imagefont.py

```diff
@@ -977,6 +977,14 @@ def test_colr_mask(self):
 
         assert_image_similar_tofile(im, "Tests/images/colr_bungee_mask.png", 22)
 
+    def test_fill_deprecation(self):
+        font = self.get_font()
+        with pytest.warns(DeprecationWarning):
+            font.getmask2("Hello world", fill=Image.core.fill)
+        with pytest.raises(TypeError):
+            with pytest.warns(DeprecationWarning):
```

Comment:
```suggestion
        with pytest.warns(DeprecationWarning):
            with pytest.raises(TypeError):
```
Testing with main, the test only fails if the checks are in this order.

Reply:
You are right, fixed. I thought I tested this, but I guess I was confused by the previous check...